src/app/modules/shared-components/components/add-content/add-content.component.html

```diff
@@ -1,23 +1,47 @@
 <alg-sub-section icon="sign-in-alt" label="Add a content">
-  <alg-input [parentForm]="newContentForm" name="title" placeholder="Enter a title to create a new child" [hasClearButton]="true"></alg-input>
+  <div class="add-content-wrapper">
+    <alg-input
+      [parentForm]="newContentForm"
+      [hasClearButton]="true"
+      (focus)="onFocus('newTitle')"
+      (blur)="onBlur()"
+      [ngClass]="{'input-light': focused === 'existingTitle'}"
+      name="newTitle"
+      placeholder="Enter a title to create a new child"
+    >
+    </alg-input>
+    <span class="label">or</span>
+    <alg-input
+      [parentForm]="newContentForm"
+      [hasClearButton]="true"
+      (focus)="onFocus('existingTitle')"
+      (blur)="onBlur()"
+      [ngClass]="{'input-light': focused === 'newTitle'}"
+      name="existingTitle"
+      placeholder="Search for existing content"
+    >
+    </alg-input>
+  </div>
+
+  <alg-message *ngIf="state === 'closed' && focused !== null" detail="Enter at least 3 characters"></alg-message>
```

Comment:
Checking a bit, I find the ngprime tooltip quite good for this usage, and only when the field is on focus (and <3char)

Reply:
Done